.teamcity/_self/projects/WPComTests.kt

```diff
@@ -404,15 +404,13 @@ private object VisualRegressionTests : BuildType({
 			notifierSettings = slackNotifier {
 				connection = "PROJECT_EXT_11"
 				sendTo = "#visual-regression-automated-tests"
-				messageFormat = verboseMessageFormat {
-	            	addBranch = false
-	            }
+				messageFormat = simpleMessageFormat()
 			}
-			buildFailedToStart = true
```

Comment:
Why drop notifications on `buildFailedToStart`, `firstSuccessAfterFailure` and `buildProbablyHanging` ?

Reply:
Was really just going for simplicity. I can add them back in

Reply:
I think those are TC defaults, and make sense to me tbh.